jobbergateapi2/tests/apps/users/test_routers.py

```diff
@@ -71,22 +71,23 @@ def test_search_without_results(client):
 @database.transaction(force_rollback=True)
 async def test_create_user(client, user_data):
     """
-    Test user creation
+    Test user creation, should be active and not superuser.
```

Comment:
"Test the creation of a user who is active, but not a superuser."

Reply:
Since is_active is default to True and is_superuser is default to False, I think maybe me more clear in the docstring about that stuff. The way you have put it, seems like we are passing the values, when we aren't